testcrate/src/bench.rs

```diff
@@ -350,7 +350,7 @@ macro_rules! impl_testio {
     }
 }
 
-#[cfg(not(feature = "no-f16-f128"))]
+#[cfg(all(f16_enabled, f128_enabled))]
 impl_testio!(float f16, f128);
```

Comment:
Since we may need one with the other disabled, these need to be separate

```rust
#[cfg(f16_enabled)]
impl_testio!(float f16);
#[cfg(f1238_enabled)]
impl_testio!(float f128);
```

Reply:
Done.